pyOCD/transport/cmsis_dap.py

```diff
@@ -381,6 +381,8 @@ def _write(self, request, data = 0):
         """
         Write a single command
         """
+        assert type(request) is int, "request is not an int"
+        assert type(data) is int, "data is not an int"
```

Comment:
When I run basic_test.py I get an assert on this because a long is passed in.  The value is still in the integer range, but due to previous operations it was converted to a long implicitly.